lib/identity_cache/configuration_dsl.rb

```diff
@@ -179,6 +179,10 @@ def disable_primary_cache_index
         self.primary_cache_index_enabled = false
       end
 
+      def should_use_cache?
```

Comment:
I think you can remove this and just include the ShouldUseCache module in [IdentityCache.included](https://github.com/Shopify/identity_cache/blob/53b41864b78d822c29277db3e9faf0050768efb1/lib/identity_cache.rb#L57-L66)